test/scripts/e2e_subs/rest.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# TIMEOUT=300
+
+date '+rest.sh start %Y%m%d_%H%M%S'
+
+set -ex
+set -o pipefail
+export SHELLOPTS
+
+WALLET=$1
+gcmd="goal -w ${WALLET}"
+ACCOUNT=$(${gcmd} account list|awk '{ print $3 }')
+
+# REST Parameters
+PUB_TOKEN=$(cat "$ALGORAND_DATA"/algod.token)
+ADMIN_TOKEN=$(cat "$ALGORAND_DATA"/algod.admin.token)
+NET=$(cat "$ALGORAND_DATA"/algod.net)
+
+function call_admin {
```

Comment:
Looks like this is a duplicate of `call`?

Reply:
Not yet, I wanted to make it handy for when the scope of these tests expand.

Reply:
tiny - since it's the same code as the regular one, `admin/pub` can simply be an argument or just have a wrapper to one function.